pennylane/operation.py

```diff
@@ -142,13 +142,18 @@ def expand_matrix(base_matrix, wires, wire_order=None):
 
     **Example**
 
-    If the wire order is identical to ``wires``, the original matrix gets returned:
+    If the wire order is ``None`` or identical to ``wires``, the original matrix gets returned:
 
     >>> base_matrix = np.array([[1, 2, 3, 4],
     ...                         [5, 6, 7, 8],
     ...                         [9, 10, 11, 12],
     ...                         [13, 14, 15, 16]])
     >>> print(expand_matrix(base_matrix, wires=[0, 2], wire_order=[0, 2]))
+    [[ 1  2  3  4]
+     [ 5  6  7  8]
+     [ 9 10 11 12]
+     [13 14 15 16]]
+    >>> print(expand_matrix(base_matrix, wires=[0, 2]))
```

Comment:
Minor: do we actually need to showcase both `None` and `wire_order=[0, 2]` cases? :thinking: Good catch removing the later occurrence nonetheless!

Reply:
Different functionality, so can't hurt.